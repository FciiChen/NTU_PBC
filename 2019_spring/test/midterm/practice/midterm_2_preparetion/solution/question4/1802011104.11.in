他轉頭，目光和領頭男子交視，隨即各自錯開。

　　仰頭望向雨幕盡頭，似乎想穿過這沉沉的雨看見自己想看見的人，又似乎在做著默默告別，頎長男子眼神中泛起淡淡疼痛和柔軟，卻一閃即逝，隨即他頭也不回，掠向敵人之中。

　　人尚未到，手腕一振。

　　「唰！」

　　地面上彈開黑色的繩索，靈活而矯健的纏上追來的奔馬，一滾一抽，最前面一匹馬慘嘶著倒地，馬上張弓搭箭的騎士猝不及防被掀翻，葫蘆似的滾下去，撞上後面的馬，那馬仰首長嘶雙蹄將抬未抬之際，雪光一閃，血影一亮如虹，一顆人頭在雨花中旋開去，隨即長刀自肘間翻出，一刀斷了當先騎士的頭，順勢一拉，齊齊斬去第二匹馬的腿，馬身轟然墜地那一刻，他已鷂子般翻身而起，撞入馬上騎士懷中，刀進，刀出！


 
　　血光爆現裡，第三個騎士也已經到了，長劍劈下風聲猛烈，蒼衍躍起，手中比尋常刀更細更薄的長刀，迎上那人的劍，刀劍相貼，「嚓」一聲。

　　馬上騎士只覺得對方的刀突然不見了，心中剛剛一喜，突然便看見一截刀尖無聲無息緊貼著自己的長劍，蛇般滑出，瞬間射爆生命的星火！

　　剎那之間，斃兩馬，殺三人！

　　血浮屠第一高手！

　　小六被領頭男子拖著奔行，猶自回頭死死盯著他閃掠如電的背影，渾身都在輕微發顫。

　　是的，整個隊伍都是老大的屬下，都該在生死之境前赴後繼，但是，不應該包括阿衍！

　　只有他知道，他是老大的親兄弟！

　　更重要的是，他是一個父親，他那三千里地一根獨苗的兒子，是那個家族最後的後代……而那孩子……那個奇異的孩子，如果沒有父親，怎麼能活成！

　　這一替，替的是兩條命，替的是血浮屠首領家族延續的最後香火。


 
　　這樣的決定，老大怎麼忍心做下？

　　他突然不掙紮了，濕漉漉的頭髮披散下來，垂在眼上，領頭男子看著少年蒼白的額，微微有些憐惜的拍拍他，解開了他的穴道。

　　「我心裡有預感，前面大概還有敵人。」領頭男子沉聲道，「如果真的是這樣，我會引開對方，你記得一定要帶……」

　　「走！」

　　他還沒說完，少年突然一抬手，一把抓過他懷中包裹扔了出去！

　　小小一團在半空中飛出一道弧線，剎那扔出好遠，雷聲隆隆裡隱約聽得包裹中細弱哭音顫顫一響，領頭男子大驚，急忙躍起去接，包裹落在手中，這才吁了一口長氣。

　　等他再回頭，少年瘦弱的身影已經掠向身後追騎之中。

　　浴血苦戰的阿衍回過頭來，望著小六，目光裡不知是喜是悲，那少年只笑笑，輕聲道：「天戰世家中人，永遠和兄弟共死。」

　　暴雨如傾，似蒼穹悲歌遼遠，末世皇朝的最後一批忠誠男兒，選擇含笑蹈死。


 
　　領頭男子抱著包袱，遠遠看著那背靠背作戰的人影，眼底泛起微光，隨即抿唇掉頭離去。

　　如果可以，他寧願選擇代替兄弟去死，但是，他不能。

　　懷中那一團輕軟無物，責任卻重如千鈞，在沒有完成自己誓言之前，他沒有理由卸下。

　　廝殺聲阻隔在雨幕和夜色之外，他奔行的身影快過閃電，遠遠的，山坳後露出一處小樹林。

　　男子眼中露出喜色，他知道樹林之後，便是終點。

　　然而那點喜色突然被凍結，他霍然轉身，低喝：「誰！」

　　黝黯的樹林寂然無聲，樹葉被風吹得唰唰響猶如鬼拍手，那一聲凝足中氣的低喝，彷彿落在空處。

　　男子皺皺眉，提足真力，按照約定向樹林之後掩映的一座茅舍傳音：「皇極之後，求見谷主，請谷主履行世代相傳密約！」

　　連呼三遍，樹林後毫無動靜，茅舍中燈光全無。


 
　　男子心中一沉，知道事情有變，立即不動聲色慢慢後退三步，環顧四周，緩緩靠上一棵地勢較高的老樹。

　　這處視野開闊，身後又有遮擋，萬一林中有敵人，也無法對他包圍攻擊。

　　在不利形勢下首先選擇最有利自己的地形，是血浮屠的必修功課。

　　男子十分謹慎，在靠上老樹之前，已經仔細觀察了樹身沒有異常，不可能對他造成傷害。

　　然而後背剛剛靠上樹身，他驀然發出一聲狂吼，一個大仰身拚命翻了出去。

　　落地時腿上鮮血淋漓。

　　樹林中人影連閃，數名灰袍老者無聲無息出現，將他包圍在正中。

　　男子面色慘然，瞪著剛才那樹的樹樁方向，那裡青苔累積，樹根盤繞，看起來沒什麼特別，然而男子瞪著那樹樁的眼神，就像看見地下鑽出了一個魔鬼。

　　地下沒有魔鬼，卻突然緩緩伸出了一隻手。

　　潔白的，不大的，看上去像是孩子的手。

　　樹林幽暗深黑，灰色的雨絲斜斜打下來，暗淡的色彩裡小手浮雕般鮮明，自蒼青的老樹身上緩緩伸出，這一幕怎麼看都有幾分詭異，男子素來穩定沉重的心，怦怦跳了起來。

　　先是手，然後是手腕……偽裝的青苔樹根被一一撥開，現出烏黑的髮頂，一個人，從樹樁的位置，鑽了出來。

　　他抬起頭。

　　男子震驚的退後一步。

　　真的是孩子。

　　不過六七歲模樣，披一件暗綠色油絨衣，看起來和那樹身顏色近似，這種顏色難看得很，穿在這孩子身上，卻讓人覺得清而雅，正如這夜雨深林幽暗泥濘污濁陰冷，他站在那裡，所有人心中卻都突然掠過一個詞——玉人。

　　明光清潤，如玉琢成。

　　不過一個孩子便已如此容色攝人，一旦長成，卻又不知該如何的顛倒眾生。

　　男子卻只抱緊懷中包袱，警惕的盯著這個孩子——他不會忘記，正是這個看來無害的小小少年，躲在這樹身之中，利用這雨夜暗林的掩護，偷襲了身經百戰的他。

　　訓練有素的血浮屠精英在密林遇險時，會習慣性的先選擇背靠大樹佔據有利地形，而正常情況下，人的視線一般都只會平齊向前而不會故意向下，他哪裡想得到在那並不粗的樹樁處，竟然會挖空藏了個孩子。

　　是巧合，還是故意安排？

　　如果是有意安排，那這孩子也太可怕——熟悉血浮屠的作戰自保方式，懂得人的習慣選擇，膽大心細，出手狠絕。

　　剛才那一刀，如果不是他應變超卓及時避過，本來是該捅在他腰眼要害的。

　　那孩子微微偏頭，有趣的瞧了瞧他，目光在他手中包袱掠過，突然淡淡道：「有些人就是蠢，何必費盡心思折損人手，像條狗似的攆在你們後面？與其千里追殺，不如守株待兔，你說，是不？」

　　男子抿了抿唇，目光向後一掠，那孩子立即道：「不用看了，你要接頭的人，已經走了。」

　　男子眼神一顫，這個山谷的主人，和先主有約定，在他前來求助聯絡之前，是絕對不會離開的，然而這林子裡鬧出這麼大動靜，後方石屋依舊毫無動靜，難道，人真的走了？

　　這麼一想心中便是絕望的一沉，然而他依舊謹慎的保持沉默，並不失措慌張，那孩子卻似能讀心一般已經輕輕笑起來，笑容清雅明潤，眼神卻晶石般冷。

　　「不相信是麼？其實很簡單，假如在你之前，已經有人帶著你們血浮屠的令牌，抱著和你懷中一樣的寶貝，求見谷主，你說，谷主大人會怎麼做？」

　　男子重重一震，駭然盯著那孩子，半晌低低道：「你怎麼會知道……」

　　屬於皇室數百年來的絕頂機密，怎麼會被這孩子知道得一清二楚？

　　「你說呢？」那孩子薄唇上的笑意，浮涼若瑟瑟秋夜裡的燈花，「這世上的秘密，只要有人知道，就遲早有被洩露的一天。」

　　男子握緊了手掌……血浮屠當中有奸細！

　　皇朝傾覆，王公盡降，忠心王朝的舊臣盡數屠戮，如今天下之大，只留下世代享受供奉，不為任何掌權者所控制的血浮屠，保留了自由之身來護持這皇朝最後一點血脈，千里追殺中多少人喪於路途，多少人拚死斷後，到得如今走到最後的寥寥幾人，阿衍、老石、三虎、小六……無一不是隊伍中最為精英、地位最高、忠誠亦最無懈可擊的成員，是他生死相托的兄弟。

　　那麼……會是誰？能是誰？

　　不能懷疑，不敢懷疑，這個念頭一旦觸及便是森冷的撕裂和無垠的陰影，如果是真的，如果那些犧牲和追隨都能有假，叫人情何以堪？

　　深深吸一口氣，男子後退一步，現在已經不是追究誰是奸細的時辰，當務之急，是完成自己的承諾。

　　他退一步，那數名灰袍老者也齊齊向前一步，動作看似平凡，男子卻精細的注意到，自己和他們之間的距離，在這一步移動過後，和原先保持得完全一樣。

　　這個發現讓他再次心中一緊，無庸置疑，對方是眼力和武力俱佳的絕頂高手，以他現在的狀態，一個也接不下，更不要說在眾人環伺之下逃脫。

　　落雨無聲，隱約聽得人緊張的呼吸粗重，當先一個灰袍老者木然抬手一指，指向那男子懷中包裹。

　　男子垂眼，聲音平靜：「……想要？拿命來換。」

　　那孩子卻笑了起來。

　　手一揮。

　　砰然一聲悶響，一團東西被擲在了林中，昏暗光線勾勒出淋漓而模糊的微紅輪廓，一時讓人看不清那是什麼，男子卻死死盯著，掩在袖子裡的雙手攥緊，指甲深入肉中。

　　那是三虎的屍體，或者說……其實已經不能算是屍體。

　　如果不是那明顯較矮的個子和腰間還剩半個的血浮屠標誌，便是三虎那個智慧卓絕狠辣明利的小女兒來認，也一定認不出。
BREAK