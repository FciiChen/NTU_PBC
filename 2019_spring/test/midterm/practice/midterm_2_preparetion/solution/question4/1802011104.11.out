血_屠 9
那_子 8
領_男 6
頭_子 6
的_影 5
子_著 4
這_子 4
一_手 3
了_來 3
他_然 3
